biblio/package.json

```diff
@@ -0,0 +1,14 @@
+{
+  "name": "TBD",
+  "version": "VERSIONED-DURING-PUBLISH",
```

Comment:
maybe it should have `"private": true`, and have the publish step remove it? to prevent accidents

Reply:
Will npm actually let you publish with a version string of `VERSIONED-DURING-PUBLISH`?

Reply:
ha, no, true enough, it wouldn't.

actually an invalid version string might break npm in a number of ways; this should probably be `0.0.0`

Reply:
This version string is good enough to run `npm version`, which is all it needs to do.